src/main/scala/org/antipathy/scoozie/Scoozie.scala

```diff
@@ -1,12 +1,12 @@
 package org.antipathy.scoozie
 
+import java.nio.file.Path
+
+import com.typesafe.config.ConfigFactory
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://app.codacy.com/app/SimonJPegg/scoozie/pullRequest?prid=3072461)